bool gait_toggle
bool hop_toggle
bool calibration_toggle
bool joystick_toggle
bool pid_toggle
float32 x               # LEFT-RIGHT movement
float32 y               # FORWARD-BACKWARD movement
float32 roll_movement
float32 pitch
float32 yaw_rate
float32 height_movement
